### PR TITLE
refactor(circularprogress): apply 2020 color palette

### PR DESCRIPTION
**Pull request by jaketrent**
approach matching linear progress.

### CODE REVIEW

File: packages/circularprogress/src/css/index.js [hunk 1]
```diff
@@ -38,20 +37,20 @@ export default {
...
+    stroke: colorsTextIcon.highOnDark
   },
   // __bg
   [`.psds-circularprogress__bg`]: {
     fill: 'transparent',
     strokeWidth: `${vars.style.strokeWidth}px`
   },
   [`.psds-circularprogress__bg.psds-theme--${themeNames.light}`]: {
-    stroke: transparentize(0.8, core.colors.gray04)
+    stroke: 'rgba(0, 0, 0, 0.15)'
```
Comment (EdwardIrby) [inline packages/circularprogress/src/css/index.js]: This is the third time I've seen this sorta usage of this color for this. Maybe it should be a core token like track/progress on lo & high

Comment (jaketrent) [reply packages/circularprogress/src/css/index.js]: Perhaps.  We have used it a few times.  In my mind, it doesn't rise to the generality level of text, icon, and border though.

Comment (EdwardIrby) [reply packages/circularprogress/src/css/index.js]: Should we maybe create a tokens package?

Comment (jaketrent) [reply packages/circularprogress/src/css/index.js]: Isn't that what core is?

**Pull request merged**